tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -15,50 +15,53 @@
+            	cp '$infile' '/runtime/data-in-ref/${infile.element_identifier}_${i}' &&
+            #end for
         #end if
-
         
         #if $config.type == 'yes':
             ## Write YAML content
             cp '${config_file}' '/runtime/config/config_user_odv_tool.yml' &&
         #end if 
 
-        /app/launchers/start-app.sh '$mode' &&
+        /app/launchers/start-app.sh &&
+        cp '/runtime/data-out/odv_collection.txt' '$output' &&
 
-        #if $mode == 'R_APP_QV':
-            cp '/runtime/data-out/odv_collection.txt' '$output' &&
-        #else:
-            cp '/runtime/data-out/demo_odv_collection.txt' '$output' &&
-        #end if
         cp -r '/runtime/data-out/' './outputs/'
     ]]></command>
     <configfiles>
         <configfile name="config_file">
-    #if $config.type == 'yes':
-            default:
-                file_name_list_raw: '${input_raw.element_identifier}'
-                operator: '${config.operator}' 
-            QV:
-                file_name_list_ref: '${input_ref.element_identifier}' 
-                odv_collection:
-                    name: "odv_collection.txt"
-                    qc_convention: '${config.qc_convention}' 
-                tool_option:
-                    subsetting: '${config.subsetting}' 
-                    plt: '${config.plt}' 
-    #end if 
+        #if $config.type == 'yes':
+                default:
+                    file_name_list_raw: 
+                        #for $i, $infile in enumerate($input_raw):
+                            #set name_raw='${infile.element_identifier}_${i}.nc'
+                        #end for
+                        - ${name_raw}
+                    operator: '${config.operator}' 
+                QV:
+                    file_name_list_ref: 
+                        #for $i, $infile in enumerate($input_ref):
+                            #set name_ref='${infile.element_identifier}_${i}.nc'
+                        #end for
+                        - ${name_ref}
+                    odv_collection:
+                        name: "odv_collection.txt"
+                        qc_convention: '${config.qc_convention}' 
+                    tool_option:
+                        subsetting: '${config.subsetting}' 
+                        plt: '${config.plt}' 
+        #end if 
         </configfile>
     </configfiles>
     <inputs>
-        <param name="input_raw" type="data" format="netcdf" label="Input raw data"/>
-        <param name="input_ref" type="data" format="netcdf" optional="true" label="Input reference data"/>
-        <param name="mode" type="select" label="Select in which mode you want to run the tool">
-            <option value="R_APP_QV">QV that creates a odv collection with raw and ref files to qualify</option>
-            <option value="DEMO">Valide with ODV demo that performes the demo</option>
-        </param>
+        <param name="input_raw" type="data" format="netcdf" multiple="true" label="Input raw data"/>
```

Comment:
Just as a note, when you need both multiple-datasets, the raw and the ref, in the same order ... I guess users can screw up here.

Maybe a repeat is better? I don't know, we can try and see if users gets confused.

Reply:
it does not necessarily need to be in the same order I think. I'll ask later on and yes let's see